boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -171,15 +190,14 @@ ifeq ($(CRD_VERSION), v1beta1)
 	find deploy/crds -name '*.yaml' | xargs -n1 -I{} yq d -i {} 'spec.**.x-kubernetes-map-type'
 	find deploy/crds -name '*.yaml' | xargs -n1 -I{} yq d -i {} 'spec.**.x-kubernetes-struct-type'
 else
-	cd pkg/apis; controller-gen crd:crdVersions=v1 paths=./... output:dir=../../deploy/crds
+	$(CONTROLLER_GEN) crd paths=./... output:dir=../../deploy/crds
 endif
-	# The artist formerly known as `operator-sdk generate k8s`:
-	cd pkg/apis; controller-gen object paths=./...
-	# Don't forget to commit generated files
+	$(CONTROLLER_GEN) object paths=./...
+
 
 .PHONY: openapi-generate
 openapi-generate:
-	find ./pkg/apis/ -maxdepth 2 -mindepth 2 -type d | xargs -t -n1 -I% \
+	find ./api/ -maxdepth 2 -mindepth 2 -type d | xargs -t -n1 -I% \
```

Comment:
Similarly, does this mean consuming repos will have to move their API packages?

Reply:
This is true, without this, you cannot use `operator-sdk create api` to add new api.